staging/src/k8s.io/apiserver/pkg/authentication/token/tokenfile/tokenfile.go

```diff
@@ -62,6 +62,10 @@ func NewCSV(path string) (*TokenAuthenticator, error) {
 		if len(record) < 3 {
 			return nil, fmt.Errorf("token file '%s' must have at least 3 columns (token, user name, user uid), found %d", path, len(record))
 		}
+		if record[0] == "" {
+			return nil, fmt.Errorf("token file '%s' must have non empty token", path)
```

Comment:
the auth layer won't actually authenticate an empty token. I'd probably warn and skip the record

Reply:
SGTM